tasks/simplify.py

```diff
@@ -0,0 +1,55 @@
+from luigi import Task, Parameter
+import json
+import os
+from util import get_logger
+from simplification import SimplifyGeometriesMapshaper, SimplifyGeometriesPostGIS
+
+
+TABLE_ID = 'tableid'
+SIMPLIFICATION = 'simplification'
+GEOM_FIELD = 'geomfield'
+FACTOR = 'factor'
+MAX_MEMORY = 'maxmemory'
+SKIP_FAILURES = 'skipfailures'
+
+OBSERVATORY_SCHEMA = 'observatory'
+SIMPLIFICATION_MAPSHAPER = 'mapshaper'
+SIMPLIFICATION_POSTGIS = 'postgis'
+
+LOGGER = get_logger(__name__)
+
+
+class SimplifyOne(Task):
+    table = Parameter()
+
+    def run(self):
+        with open(os.path.join(os.path.dirname(__file__), 'simplifications.json')) as file:
+            tables = json.load(file)
+
+            if self.table in tables:
+                table = tables.get(self.table)
+                LOGGER.info("Simplifying %s (%s) using %s", self.table, table.get(TABLE_ID), table.get(SIMPLIFICATION))
+
+                simplification = table.get(SIMPLIFICATION)
```

Comment:
In this cases, where we expect to be a value, I would replace all `.get` calls with array accesses `[]`. A missing parameter is an error here, is it not?